manifests/quick-start/sso/overlays/argo-server-sa.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-server
+  annotations:
+    # The rule is an expression used to determine if this service account should be used. 
+    # * `groups` - an array of the OIDC groups
+    # * `iss` - the issuer
+    # * `sub` - the subject (typically the username)
+    # Must evaluate to a boolean. 
+    # If you want an account to be the default to use, this rule can be "true".
+    # Details of the expression language are available in
+    # https://github.com/antonmedv/expr/blob/master/docs/Language-Definition.md.
+    workflows.argoproj.io/rbac-rule: "'authors' in groups"
+    # The precedence is used to determine which service account to use whe
+    # Precedence is an integer. It may be negative. If omitted, it defaults to "0".
+    # Numerically higher values have higher precedence (not lower, which maybe counter-intuitive to you).
+    # If two rules match and have the same predence, then which one used will be arbitrary.
```

Comment:
If two rules match with same precedence then don't do arbitrary - sort by name or issue an error & don't allow access.  I know not as neat to code but far more understandable to admin-user.